jscomp/core/j.ml

```diff
@@ -159,7 +159,7 @@ and expression_desc =
   *)
   | Number of number
   | Object of property_map
-  | Undefined
+  | Undefined of bool (* true means: unit *)
```

Comment:
Might be nice to use an inline record here

```ocaml
  | Undefined of {isUnit: bool}
```

Reply:
Good idea.